Wi-Fi Zone/src/main/java/by/gsu/com/Values.java

```diff
@@ -0,0 +1,88 @@
+package by.gsu.com;
+
+
+import java.io.Serializable;
+
+public class Values implements Serializable {
+
+    private Double transmitterPower;
+    private Double transmitAntennaGain;
+    private Double receiveAntennaGain;
+    private Double receiverSensitivity ;
+    private Double signalLoss1;
+    private Double signalLoss2;
+    private Double frequency;
+    private double fsl;

```

Comment:
Where do you set value for this field? There is nothing in constructor. Is this programm realy works?